contracts/IssuanceFactory.sol

```diff
@@ -203,9 +200,13 @@ contract IssuanceFactory is CurrencyFactory {
 
     require(ERC20(_token).transfer(msg.sender, ccAmount));
     require(ERC20(clnAddress).transfer(msg.sender, clnAmount));
+
+		Ownable(marketMakerAddress).requestOwnershipTransfer(msg.sender);
```

Comment:
odd, that should have been there before the change...

Reply:
yeah, thought that too 😅